atlas/packages/app/src/components/Navbar/Navbar.tsx

```diff
@@ -1,60 +1,87 @@
 import React, { useState } from 'react'
-import { navigate, Link, RouteComponentProps } from '@reach/router'
+import { RouteComponentProps, Link, navigate } from '@reach/router'
 
-import routes from '@/config/routes'
 import { Icon } from '@/shared/components'
-import { Header, NavigationContainer, StyledIcon, StyledSearchbar, CancelButton, Logo } from './Navbar.style'
+import routes from '@/config/routes'
+import {
+  Header,
+  NavigationContainer,
+  StyledIcon,
+  StyledSearchbar,
+  SearchbarContainer,
+  CancelButton,
+  CancelButtonContainer,
+  Logo,
+} from './Navbar.style'
 
-const Navbar: React.FC<RouteComponentProps> = () => {
-  const [search, setSearch] = useState('')
-  const [isSearching, setIsSearching] = useState(false)
+type NavbarProps = RouteComponentProps
 
-  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
-    setSearch(e.currentTarget.value)
-  }
+const Navbar: React.FC<NavbarProps> = () => {
+  const [search, setSearch] = useState('')
+  const [focus, setFocus] = useState(false)
 
   const handleKeyPress = (e: React.KeyboardEvent<HTMLInputElement>) => {
-    if (e.key === 'Enter' || (e.key === 'NumpadEnter' && search.trim() !== '')) {
+    if (e.key === 'Enter' || (e.key === 'NumpadEnter' && search.trim())) {
```

Comment:
Auto-format grouped them, I changed it to be:
1. User pressed `Enter` **or** `NumpadeEnter`
2. **And** search isn't empty

Reply:
FYI, this probably wasn't because of the formatter. My guess is your original condition looked like so: `e.key === 'Enter' || e.key === 'NumpadEnter' && search.trim()`. However, the `AND` operator has higher precedence than the `OR` so when formatted added grouping around `e.key === 'NumpadEnter' && search.trim()` it was only to improve the readability of what the condition actually represents, the logic remained intact.